### PR TITLE
Display posts

### PR DESCRIPTION
**Pull request by NGFuture**

### CODE REVIEW

**Comment by NGFuture** [comment]
Please merge my part. Please do not edit it as I am making a lot of updates and going to push a new version shortly. Thank you.
(to make it work you need new package, please run "npm install" after merging

**Review by MunirNuristani — APPROVED** [review]
commiting changes

**Comment by NGFuture** [comment]
To resolve merge conflict I've used package.json from current master repo.

File: package.json [hunk 1]
```diff
@@ -11,11 +11,10 @@
     "@chakra-ui/core": "^0.8.0",
     "bootstrap": "^5.1.3",
     "css-loader": "^6.5.1",
-    "firebase": "^9.6.2",
+    "firebase": "^8.10.0",
```
Comment (CodergirlVS) [inline package.json:14]: The new changes in package.json show "firebase": "^8.10.0",  we need it to be 9.6.2.

Comment (NGFuture) [reply package.json:14]: I am not sure where this 8.10 comes from... As I have "firebase": "^9.6.2",

**Review by CodergirlVS — APPROVED** [review]
There are no conflicts

File: package.json (hunk 1 above)
Comment (CodergirlVS) [reply package.json:14]: I will push this merge and then can you push again with the 9.6.2. ?

**Review by CodergirlVS — APPROVED** [review]

**Pull request merged**